common/schema/types.go

```diff
@@ -91,6 +91,9 @@ type FlowMessage struct {
 	DstAS     uint32
 	GotASPath bool
 
+	SrcNetMask uint32
+	DstNetMask uint32
```

Comment:
This can be `uint8`.

Reply:
Done. Had it on 32 initially due to the sflow data type.